integration-cli/docker_cli_build_test.go

```diff
@@ -7371,7 +7369,13 @@ WORKDIR /foo/bar
 	c.Assert(err, checker.IsNil, check.Commentf("Output: %s", out))
 
 	out, _ = dockerCmd(c, "inspect", "--format", "{{ json .Config.Cmd }}", image)
-	c.Assert(strings.TrimSpace(out), checker.Equals, `["sh"]`)
+
+	// The Windows busybox image has a blank `cmd`
```

Comment:
hm, two questions;

- is there a reason the `busybox` image on Windows does not have a default `cmd`? Seems rather inconsistent
- by checking for `null`, is this still properly validating that setting a `WORKDIR` does not overwrite the original `CMD`?

Should we use a different image that originally _had_ a `CMD` instead? (perhaps the `hello-world` image works?)